### ISSUE TITLE
How do I set English as default? 

### ISSUE THREAD

**Issue by deonis1** [fardjad/node-llmatic#30]
For some reason when I send a request to the model I get response in random language, Korean, Mandarin etc and only sometimes in English. Do you know how to set English as default? My model is /ggml-vic13b-q5_1.bin
curl -X 'POST'   'http://127.0.0.1:3000/v1/chat/completions'   -H 'accept: application/json'   -H 'Content-Type: application/json'   -d '{
  "model": "string",
  "messages": [
    {
      "role": "system",
      "content": "Hello"
    }, {"role":"user", "content":"Hi"}
  ],
  "temperature": 1,
  "max_tokens":50,
  "stream": false
}'

**Comment by fardjad** [comment]
I think this is more related to the model and not LLMatic itself, but I'd try to lower the temperature

**Comment by deonis1** [comment]
After some playing with the model this seems to work well with Vicunia 13b model (https://huggingface.co/vicuna/ggml-vicuna-13b-1.1/tree/main) : 

curl -X 'POST' \
  'http://127.0.0.1:3000/v1/chat/completions' \
  -H 'accept: application/json' \
  -H 'Content-Type: application/json' \
  -d '{
  "model": "string",
  "messages": [
    {
      "role": "system",
      "content": "You are assistant ",
      "name": "string"
    }, 
    {
      "role": "user",
      "content": "Tell me about yourself",
      "name": "string"
    }
  ],
  "temperature": 1,
  "max_tokens": 2000,
  "stream": false
}
'
 I do not know why but I need to leave a space at the end of "You are assistant ". If I do not, it returns nothing. I will try different Lama models perhaps it is just vicunia doing funny things. Thank you for your help! Do you know by any chance how I can use  GPU with the API?

**Comment by deonis1** [comment]
this is my llmatic.config.json file: {
  "enableLogging": false,
  "nParts": 1,
  "nGpuLayers": 0,
  "f16Kv": false,
  "logitsAll": false,
  "vocabOnly": false,
  "seed": 0,
  "useMlock": true,
  "embedding": true,
  "useMmap": true,
  "nCtx": 4096,
  "nThreads": 16,
  "nTokPredict": 32768,
  "topK": 40,
  "topP": 0.95,
  "temp": 0.5,
  "repeatPenalty": 1.1,
  "modelPath": "./models/ggml-vic13b-q5_1.bin"
}

**Comment by fardjad** [comment]
Under the hood, LLMatic default adapter uses llama-node which offers node-bindings for llama.cpp. You might need to create a custom adapter and compile llama-node to make GPU inference work:

https://llama-node.vercel.app/docs/cuda

You might also need to set the nGpuLayers parameter. Please have a look at llama.cpp docs for more information:

https://github.com/ggerganov/llama.cpp

Since this issue is not related to LLMatic itself, I'm going to close it for now. Please feel free to reopen if you run into an issue with LLMatic itself.
